
(3, 197)

Performance of addy-finetuned model on addy dataset: 0.99

Performance of addy-finetuned model on sub dataset: 0.545

Performance of addy-finetuned model (plus 10 examples on sub) model on sub dataset: 0.865

Performance of addy-finetuned model on spain dataset: 0.0

Performance of addy-finetuned model (plus 10 examples on spain) model on spain dataset: 0.23880597014925373

Performance of addy-finetuned model on div dataset: 0.91

Performance of addy-finetuned model (plus 10 examples on div) model on div dataset: 1.0

Performance of addy-finetuned model on mex dataset: 0.365

Performance of addy-finetuned model (plus 10 examples on mex) model on mex dataset: 0.41

Performance of addy-finetuned model on mult2x1 dataset: 0.925

Performance of addy-finetuned model (plus 10 examples on mult2x1) model on mult2x1 dataset: 0.965

Performance of addy-finetuned model on faith dataset: 0.1

Performance of addy-finetuned model (plus 10 examples on faith) model on faith dataset: 0.61

Performance of addy-finetuned model on science dataset: 0.12

Performance of addy-finetuned model (plus 10 examples on science) model on science dataset: 0.6

Performance of sub-finetuned model on addy dataset: 0.915

Performance of sub-finetuned model (plus 10 examples on addy) model on addy dataset: 0.93

Performance of sub-finetuned model on sub dataset: 0.975

Performance of sub-finetuned model on spain dataset: 0.0

Performance of sub-finetuned model (plus 10 examples on spain) model on spain dataset: 0.23880597014925373

Performance of sub-finetuned model on div dataset: 0.895

Performance of sub-finetuned model (plus 10 examples on div) model on div dataset: 1.0

Performance of sub-finetuned model on mex dataset: 0.29

Performance of sub-finetuned model (plus 10 examples on mex) model on mex dataset: 0.34

Performance of sub-finetuned model on mult2x1 dataset: 0.915

Performance of sub-finetuned model (plus 10 examples on mult2x1) model on mult2x1 dataset: 0.935

Performance of sub-finetuned model on faith dataset: 0.105

Performance of sub-finetuned model (plus 10 examples on faith) model on faith dataset: 0.605

Performance of sub-finetuned model on science dataset: 0.12

Performance of sub-finetuned model (plus 10 examples on science) model on science dataset: 0.6

Performance of spain-finetuned model on addy dataset: 0.64

Performance of spain-finetuned model (plus 10 examples on addy) model on addy dataset: 0.905

Performance of spain-finetuned model on sub dataset: 0.1

Performance of spain-finetuned model (plus 10 examples on sub) model on sub dataset: 0.79

Performance of spain-finetuned model on spain dataset: 0.24378109452736318

Performance of spain-finetuned model on div dataset: 0.525

Performance of spain-finetuned model (plus 10 examples on div) model on div dataset: 0.92

Performance of spain-finetuned model on mex dataset: 0.245

Performance of spain-finetuned model (plus 10 examples on mex) model on mex dataset: 0.295

Performance of spain-finetuned model on mult2x1 dataset: 0.505

Performance of spain-finetuned model (plus 10 examples on mult2x1) model on mult2x1 dataset: 0.885

Performance of spain-finetuned model on faith dataset: 0.405

Performance of spain-finetuned model (plus 10 examples on faith) model on faith dataset: 0.65

Performance of spain-finetuned model on science dataset: 0.45

Performance of spain-finetuned model (plus 10 examples on science) model on science dataset: 0.6

Traceback (most recent call last):
  File "/mnt/c/Users/riopo/My Drive/B. Academics/GPI/Generalization/transfer-scaling/pairwise_eval.py", line 24, in <module>
    correct, incorrect = eval_model_on_dataset(model_finetuned_on_first_task_name, second_task_name)
  File "/mnt/c/Users/riopo/My Drive/B. Academics/GPI/Generalization/transfer-scaling/gpt_eval.py", line 24, in eval_model_on_dataset
    response = client.completions.create(
  File "/usr/local/lib/python3.10/dist-packages/openai/_utils/_utils.py", line 271, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/openai/resources/completions.py", line 506, in create
    return self._post(
  File "/usr/local/lib/python3.10/dist-packages/openai/_base_client.py", line 1167, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "/usr/local/lib/python3.10/dist-packages/openai/_base_client.py", line 856, in request
    return self._request(
  File "/usr/local/lib/python3.10/dist-packages/openai/_base_client.py", line 947, in _request
    raise self._make_status_error_from_response(err.response) from None
openai.NotFoundError: Error code: 404 - {'error': {'message': 'The model `ft:davinci-002:personal:div-ft:8pmf8d9M` does not exist or you do not have access to it.', 'type': 'invalid_request_error', 'param': None, 'code': 'model_not_found'}}
